### DISCUSSION TITLE
Missing "CO2" sensor category

### DISCUSSION THREAD

**Discussion by andras-tim** [custom-components/ble_monitor#693] [General]
I'm looking for CO2 sensors, and I have read you support the "Sensirion SCD4x" sensor, but I can't find the category and the device in the [catalog](https://custom-components.github.io/ble_monitor/).

_I see, the Sensirion's release was 17 hours ago - so maybe this issue was a bit fast, sry - but I don't know this project supports any other CO2 sensors, or not._

**Comment by Ernst79** [comment]
Docs were missing, I see. They are added now.

https://custom-components.github.io/ble_monitor/by_brand#sensirion

**Reply by andras-tim** [reply]
Thank you!

( https://custom-components.github.io/ble_monitor/by_property#co2 :heart: )
